api/test/wfl/tools/endpoints.clj

```diff
@@ -10,7 +10,7 @@
   "The WFL server URL to test."
   [& parts]
   (let [url (util/de-slashify (env/getenv "WFL_WFL_URL"))]
-    (str/join "/" (cons url parts))))
+    (str/join "/" (cons url (map #(util/trim-slashes %) parts)))))
```

Comment:
@rexwangcc: This `wfl-url` is gettin craazy.
We shouldn't even need `de-slashify` since we control `WFL_WFL_URL`.
Make sure that `WFL_WFL_URL` doesn't end in `/` wherever we set it.
Then just `(str/join "/" (cons (env/getenv "WFL_WFL_URL") parts))`
What am I missing?

Reply:
@tbl3rd the trim-slashes was meant for the url parts, if they are written in with slashes, the join inserts double slashes like workload//:uuid//workflows

Reply:
Why would they have /s?
Doc, it hurts when I do this.
Then don't do that.

Reply:
FWIW, I don't think the previous implementation really was breaking anything anyway. Was just something Ed pointed out.